pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -155,10 +155,12 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 	configurationReconciler := knative.NewConfigurationReconciler(r.Client)
 	routeReconciler := knative.NewRouteReconciler(r.Client)
 
+	// Initialize conditions
+	kfsvc.Status.InitializeConditions()
 	// Reconcile configurations
 	desiredDefault := resourceBuilder.CreateKnativeConfiguration(constants.DefaultConfigurationName(kfsvc.Name),
 		kfsvc.ObjectMeta, &kfsvc.Spec.Default)
-
+	log.Info("reconciling default configuration", "name", desiredDefault.Name)
```

Comment:
these logs will be lost in the refactor.

Reply:
merged your change now